### DISCUSSION TITLE
Spare PCB Boards for UK

### DISCUSSION THREAD

**Discussion by chunkysteveo** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#291] [General]
**27th June 2022 - ALL GONE**

![20220601_085403](https://user-images.githubusercontent.com/14663002/171357009-a776dd0e-3e8b-4945-8ee1-d626529e4ddd.jpg)

I have two (maybe three) boards available to ship in the UK. Anyone UK based interested in a pcb board? Let's say £2 shipped anywhere in the UK as first class letter?

**Comment by dwoosnam** [comment]
I'd be interested if they're still available please @chunkysteveo ?

**Comment by chunkysteveo** [comment]
Hey @dwoosnam yes I still have a spare one.

**Reply by dwoosnam** [reply]
How do we do this? I can't see a DM feature :(

**Reply by chunkysteveo** [reply]
steve @ chunkymedia co uk ----- send me an email with your address and i can send you my PayPal link or similar?! You want an 8channel level shifter too??

**Comment by chunkysteveo** [comment]
My paypal email is: steve@chunkymedia.co.uk @dwoosnam

**Comment by GoblinRules** [comment]
Hi, do you still have one or are you all out now?

**Reply by chunkysteveo** [reply]
Hey @GoblinRules  - There's been some interest on the boards - but I do still have one spare board left if you wanted it?

**Reply by GoblinRules** [reply]
Yes please that would be great, should I use the above paypal address?
Also do you have any links on where to get the JST-SM Housing Connectors I seem to be struggling to find the right ones.

Cheers

**Reply by chunkysteveo** [reply]
Hiya,

Yes, you can use the above email for my PayPal, and also to email me your postal address. Thanks.

For the JST connectors, I used used the wires, and soldered them straight to the board. I got these, if you are a 4-pin pump person - https://s.click.aliexpress.com/e/_DmhOcWT Otherwise you will need a 6pin version of similar.

Steve

**Reply by chunkysteveo** [reply]
Here's a 6pin listing, go for the 10cm ones, you want them short as possible: https://s.click.aliexpress.com/e/_DcI0qj5

**Reply by GoblinRules** [reply]
Ok payment sent and addresss.
Great thanks for the links. will check them out, may just go the older route, saves the wait.

**Reply by GoblinRules** [reply]
thank you again.

**Reply by chunkysteveo** [reply]
Thanks for the payment @GoblinRules  - you sent too much to me, is that right/OK?

**Reply by GoblinRules** [reply]
Yes of cause, paid for the board (based on the quotes I saw they were) &  postage to me. (if its short let me know)

**Reply by chunkysteveo** [reply]
Was only after the postage - £2, but much appreciated (not short!)! Do you have the 4pin or 6pin pump?

**Reply by GoblinRules** [reply]
6pin

**Reply by GoblinRules** [reply]
I have the Hawaii 2021 Model but the serial matches that of the 2021 Miami.

**Reply by chunkysteveo** [reply]
Ahhh not to worry then! I was going to throw in some 4pin wires for you, but they won't work on a 6pin(!) 

I've just got back from my run and via the post office, so i'll post your PCB board out tomorrow if that's OK?!

Steve

**Reply by GoblinRules** [reply]
No worries, thanks for the thought.
No at a problem at all

**Reply by chunkysteveo** [reply]
Just posted it out @GoblinRules  - hopefully with you tomorrow.

**Reply by GoblinRules** [reply]
Received it today, thanks very much.

**Reply by chunkysteveo** [reply]
Awesome! Glad it arrived!!

**Comment by chunkysteveo** [comment]
ALL BOARDS FROM ME ARE NOW SOLD!

If anyone here in the UK is after PCBs made, I'm happy to order and post them in from China, and can then distribute them here within the UK. Would be worth checking out V2 and V1 of the PCB boards too.

**Reply by wyliecoyoteuk** [reply]
I have some V1 boards if anyone wants one.
also some 6 wire JST connectors

**Reply by sponte** [reply]
Hey, interested in the PCBs and 6-wire connectors. Still available?

**Reply by wyliecoyoteuk** [reply]
Yes, I have 2 v1 boards spare.
I think I still have some connectors, but I will have to find them.

**Reply by wyliecoyoteuk** [reply]
Found them! I have 2 pairs of JST connectors, and 6 v1 PCBs available.
Postage to **UK** 1st class £2 for bare PCB
JST connectors need to be sent as a large letter, so £3 for a board and pair of connectors with wires.

Other countries depending on postage costs

**Comment by GoblinRules** [comment]
Hi @chunkysteveo to confirm these are V1 boards right?

**Reply by chunkysteveo** [reply]
Hiya! Yes, the board you got was the original 'V1' board.

**Reply by GoblinRules** [reply]
Awesome, thank you.
